### ISSUE TITLE
[Eclipse-ThirdParty][A11yMAS][Accessibility][Keyboard][MAS2.4.3][Eclipse][Wipro]:Keyboard focus order is incorrect when show properties window opens in editor.

### ISSUE THREAD

**Issue by WiproPSAccTest** [Microsoft/azure-tools-for-java#1415]
**Repro Steps:**

1. Open and Start Eclipse.
2. Create a new dynamic web project.
3. In "Azure Explorer" , select the created app under web apps and use shift+F10 to open the context menu.
4. Navigate to and select "Show properties" menu item  and hit enter then observe the keyboard focus

**Expected Result:**
After hitting tab, Keyboard focus should be on show properties window

**Actual Result:**
Keyboard focus is in New button which is present in top pf the page

**Platform:**
**Eclipse Version:**
Eclipse Java EE IDE for Web Developers.
Version: Oxygen.2 Release (4.7.2)
Build id: 20171218-0600
**OSBuild Version**: Version 1709(Os Build 16299.192)

![after hitting webapp then show properties keyboard focus is going to new button](https://user-images.githubusercontent.com/22956130/35716947-079ade22-0802-11e8-93a1-9dea57607fb7.jpg)

**Comment by jdneo** [comment]
Fixed

**Comment by Eskibear** [comment]
fixed in #1605 , will be in next release

**Comment by Eskibear** [comment]
already released in v3.6.0

**Comment by WiproPSAccTest** [comment]
Verified the issue in V 3.6.0.
Issue got fixed

**Comment by vscjavaci** [comment]
Track internal issue by AB#1749873
